pkg/scheduler/framework/runtime/framework.go

```diff
@@ -722,22 +722,21 @@ func (f *frameworkImpl) RunFilterPlugins(
 	state *framework.CycleState,
 	pod *v1.Pod,
 	nodeInfo *framework.NodeInfo,
-) framework.PluginToStatus {
-	statuses := make(framework.PluginToStatus)
+) (status *framework.Status) {
```

Comment:
Named return values are recommended only for short functions (couple of lines)

Reply:
Updated.